### ISSUE TITLE
Slow navigation

### ISSUE THREAD

**Issue by belf88** [nextcloud/deck#3710]
Hi everyone, I’m trying to implement a management software on Nextcloud through the Deck application.

I have set up 20 boards with over 300 tasks and attached pdf files but the system is really slow. it is currently installed on a service of a German server company.
Usually Nextcloud freezes and the system becomes unusable.

Do you have to suggest me a correct version of Nextcloud and the Deck application to avoid usability problems?

Thanks

**Comment by neerro** [comment]
No it's impossible to speedup your system as the nextcloud and all its apps a slow solution in principal.

**Comment by putt1ck** [comment]
> Hi everyone, I’m trying to implement a management software on Nextcloud through the Deck application.

Sounds interesting :)

> I have set up 20 boards with over 300 tasks and attached pdf files but the system is really slow. it is currently installed on a service of a German server company. Usually Nextcloud freezes and the system becomes unusable.
> 

What version of NC and Deck? What resource is available to the server/database? In what circumstances is it slow? E.g. all the time, only when accessing Deck, only when accessing boards with a lot of cards, etc.

**Comment by neerro** [comment]
> No, it's impossible to speedup your system as the nextcloud and all its apps a slow solution in principal.

Why does this comment has been marked as spam? This is my personal opinion and instead of that please explain me why it's not only so...

**Comment by stefan-niedermann** [comment]
@neerro because your comment does not provide any help, new information, a solution or constructive feedback. Everyone gets notified about your personal opinion which you posted in a destructive way that does not help the developer, the issue reporter or anyone else.

Please feel free to post rants in the [corresponding category of the help forum](https://help.nextcloud.com/tags/c/general/127/rant), but this is nothing that helps for the actual development.

**Comment by corpix** [comment]
nextcloud 32.0.3
deck 1.16.2

My context:
- anount of cards ~100 (not more than 100)
- "use bigger card view" off
- clicking first card open time might be better, but fine
- second card and other are just getting worse and worse
- in my case card open time are around 7 seconds

I have done some profiling, there are a lot of reflows (see screenshot) 
<img width="2320" height="751" alt="Image" src="https://github.com/user-attachments/assets/195171b7-8c49-488f-bacd-09b90e839f63" />

**Comment by nemphys** [comment]
Indeed, any interaction with any card (eg. click) seems to cause massive forced reflows, as if all deck cards (more than 200 in my case) are redrawn.

<img width="1550" height="1748" alt="Image" src="https://github.com/user-attachments/assets/41fb489e-1abe-4320-a78d-515e877b224d" />
